ansible/roles/gelato-ha-installer/tasks/main.yml

```diff
@@ -0,0 +1,42 @@
+# Copyright 2018 The OpenSDS Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Note: It is assumed that the Kubernetes master node has all the prereqs for running Multicloud services. 
+#       If any dependency is found to be missing, it will be opened and fixed as issues
+
+---
+- name: create gelato-ha work directory if it doesn't exist
+  file:
+    path: "{{ item }}"
+    state: directory
+    mode: 0755
+  with_items:
+    - "{{ gelato_work_dir }}"
+    - "{{ gelato_tarball_dir }}"
+
+- name: include scenarios/release.yml when installing from release
+  include: scenarios/release.yml
+  when: install_gelato_ha_from == "release" or install_gelato_ha_from == "container"
+
+- name: Execute the kubectl yamls.
+  shell: "{{ item }}"
+  args:
+    chdir: "{{ gelato_work_dir }}/multi-cloud/K8s"
+  with_items:
+    - bash "k8s-mongo.sh"
```

Comment:
scripts missing

Reply:
The script will be part of multi-cloud repo, here I am downloading the code on the Kubernetes Host and executing the scripts there. These scripts are part of multi-cloud/K8s . Please check